src/view/com/lightbox/Lightbox.tsx

```diff
@@ -49,16 +49,9 @@ export function Lightbox() {
     [permissionResponse, requestPermission, _],
   )
 
-  if (!activeLightbox) {
-    return null
```

Comment:
we don't want that here anymore so that the props flow down and trigger leave animation